src/main/java/com/mael/payroll/paymentTypes/HourlyPayment.java

```diff
@@ -10,4 +13,12 @@ public HourlyPayment(double hourlyRate) {
     public double getHourlyRate() {
         return hourlyRate;
     }
+
+    public TimeCard getTimeCard() {
+        return timeCard;
+    }
+
+    public void addTimeCard(TimeCard timeCard) {
```

Comment:
The funny thing is, I first used set, then I changed to add. No problem with changing it back to set again. I guess it made sense from the transaction point of view, but you're right that here set is prolly a better option.